lib/testing/test_runner.sh

```diff
@@ -26,7 +26,7 @@ GENERATED_NAME_PREFIX="xdptest"
 ALL_TESTS=""
 TEST_PROG_DIR="${TEST_PROG_DIR:-$(dirname $0)}"
 
-NEEDED_TOOLS="ethtool ip ping tc tcpdump"
+NEEDED_TOOLS="capinfos ethtool ip ping tc tcpdump"
```

Comment:
Which package is this coming from? Please add that to the RPM spec and .travis.yml

Reply:
Its part of wireshark, will add the package.